README.md

```diff
@@ -18,32 +18,92 @@ This oracle handles the following cases among others (let's say that our pair is
 
-Run test: `forge test`
+## MorphoChainlinkOracleV2Factory
+
+The `MorphoChainlinkOracleV2Factory` is a factory that deploys `MorphoChainlinkOracleV2` instances.
+
+The factory facilitates the easy creation and indexing of `MorphoChainlinkOracleV2` oracles. This factory contract streamlines the process of deploying new oracles, catering to various market pairs without the need for deploying individual contracts manually for each pair.
+
+### Usage
+
+The Ethereum Mainnet address of this factory is [0x3A7bB36Ee3f3eE32A60e9f2b33c1e5f2E83ad766](https://etherscan.io/address/0x3a7bb36ee3f3ee32a60e9f2b33c1e5f2e83ad766#code).
+
+To create a new `MorphoChainlinkOracleV2` oracle using the factory, call the `createMorphoChainlinkOracleV2` function with the following parameters:
+
+- `baseVault`: The ERC4626 token vault for the base asset.
+- `baseVaultConversionSample`: A sample size for converting base vault units.
+- `baseFeed1`, `baseFeed2`: Chainlink-interface-compliant data feeds for the base asset.
+- `baseTokenDecimals`: Decimal precision of the base asset.
+- `quoteVault`: The ERC4626 token vault for the quote asset.
+- `quoteVaultConversionSample`: A sample size for converting quote vault units.
+- `quoteFeed1`, `quoteFeed2`: Chainlink-interface-compliant data feeds for the quote asset.
+- `quoteTokenDecimals`: Decimal precision of the quote asset.
+- `salt`: A unique identifier to create deterministic addresses for deployed oracles.
 
-## Deploy an Oracle
+### Deploy an Oracle
 
-For example, to deploy a `ChainlinkOracle` on the Ethereum mainnet for the sDAI/USDC market, run:
+To deploy a `MorphoChainlinkOracleV2` on the Ethereum mainnet, use the `MorphoChainlinkOracleV2Factory` to create a new oracle.
 
-```bash
-forge create src/ChainlinkOracle.sol:ChainlinkOracle --constructor-args "0x83F20F44975D03b1b09e64809B757c47f942BEeA" "0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9" "0x0000000000000000000000000000000000000000" "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6" "0x0000000000000000000000000000000000000000" "1000000000000000000" "18" "6" --rpc-url https://eth-mainnet.g.alchemy.com/v2/<alchemy-key> --from <deployer-address> --ledger
-``````
+Below are the arguments to fill for the creation of the WETH/USDT oracle:
 
-And for the rETH/WETH market, run:
+- `baseVault`: 0x0000000000000000000000000000000000000000,
+- `baseVaultConversionSample`:1,
+- `baseFeed1`: 0x0000000000000000000000000000000000000000,
+- `baseFeed2`: 0x0000000000000000000000000000000000000000,
+- `baseTokenDecimals`:18,
+- `quoteVault`:0x0000000000000000000000000000000000000000,
+- `quoteVaultConversionSample`:1,
+- `quoteFeed1`: 0xEe9F2375b4bdF6387aa8265dD4FB8F16512A1d46,
+- `quoteFeed2`: 0x0000000000000000000000000000000000000000,
+- `quoteTokenDecimals`:6,
+- `salt`: <user-defined value used to make the address unique>,
```

Comment:
```suggestion
```json
{
  "baseVault": "0x0000000000000000000000000000000000000000",
  "baseVaultConversionSample": 1,
  "baseFeed1": "0x0000000000000000000000000000000000000000",
  "baseFeed2": "0x0000000000000000000000000000000000000000",
  "baseTokenDecimals": 18,
  "quoteVault": "0x0000000000000000000000000000000000000000",
  "quoteVaultConversionSample": 1,
  "quoteFeed1": "0xEe9F2375b4bdF6387aa8265dD4FB8F16512A1d46",
  "quoteFeed2": "0x0000000000000000000000000000000000000000",
  "quoteTokenDecimals": 6,
  "salt": "user-defined 32 bytes used to make the address unique",
```

```